src/runtime_src/core/pcie/linux/device_linux.cpp

```diff
@@ -223,4 +223,31 @@ update_SC(const std::string& file) const
   std::cout << "TO-DO: update_SC\n";
 }
 
+unsigned long
+device_linux::
+get_ip_layoutsize(uint64_t _deviceID) const
```

Comment:
deviceID is no longer part of the device_* class objects.  The device itself is created from a device index.